### DISCUSSION TITLE
【灌水】关于等级

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#1429] [其他]
绳网等级是怎么判定的？怎么突然变4级了 

![image](https://github.com/user-attachments/assets/88aade6a-77e9-4cc1-a8e5-a2ff69cfc53a)

各位绳斤都多少级了？

**Comment by Dave-12138** [comment]
5级，应该是因为我有5个repositores（仓库）是公开的

**Comment by capacitor1** [comment]
>  Reply::5级，应该是因为我有5个repositores（仓库）

好好，有道理 马上去发点史山代码doge

**Comment by Todysheep** [comment]
竟然是这样，马上去发史山（

**Comment by inkluv** [comment]
各位别发史山，直接找项目fork也算（

**Comment by 1luik** [comment]
3

**Comment by rong-xiaoli** [comment]
👈史山代码拥有者
